app/models/cloud_network.rb

```diff
@@ -40,7 +38,7 @@ class CloudNetwork < ApplicationRecord
     end
   end
 
-  virtual_total :total_vms, :vms
+  virtual_total :total_vms, :vms, :uses => :vms
```

Comment:
>we need to fix this

I meant to write: you are right. it is not picked up. we'll add now and correct in the future